cmd/root.go

```diff
@@ -123,7 +123,7 @@ func init() {
 	}
 
 	RootCmd.PersistentFlags().String("log", "info", "The log level to output")
-	RootCmd.PersistentFlags().String("log-file", "", "File to output logs to")
+	RootCmd.PersistentFlags().String("log-file", "ficsit-cli.log", "File to output logs to")
```

Comment:
This could be replaced with `filepath.Clean(filepath.Join(baseLocalDir, "ficsit", "ficsit-cli.log"))` which will then put it in the users %APPDATA% dir, which makes more sense than wherever the CLI is ran from.